logsink/logsink.go

```diff
@@ -0,0 +1,62 @@
+package logsink
+
+import (
+	"context"
+)
+
+// Logs is a list of logs
+type Logs []*Log
+
+// Log is a record or event that has occurred
+type Log struct {
+	TraceID        string                 `json:"traceId,omitempty"`
+	SpanID         string                 `json:"spanId,omitempty"`
```

Comment:
`[]byte` probably would be more idiomatic for "byte sequence". `string` probably works too provided that there is no expectation that this should contain valid runes. A comment saying that these are byte sequences is probably sufficient. Do note that not all byte sequences are representable as JSON strings so JSON serialization/deserialization needs to be custom (and data model does not define this).

Reply:
any example that you have in mind for byte sequences which cannot be represented as JSON strings?

Reply:
added comment

Reply:
JSON is a Unicode text, often UTF-8. Not all byte sequences are valid Unicode sequences. See e.g. https://en.wikipedia.org/wiki/UTF-8#Invalid_byte_sequences
Invalid byte sequences may trigger errors or be silently skipped, etc, depending on JSON implementation.